### ISSUE TITLE
Object creations should be placed on same line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#533]
We should report a violation in case the `new` keyword of an object creation is placed on a different line than the type itself.

For example, following code should report a violation:
```c#
var xyz =
          new
             object();
```
while following code should NOT report a violation:
```c#
var xyz = new object();
```
